web/ui/dashboard/src/app/private/pages/project/apps/apps.component.html

```diff
@@ -1,14 +1,26 @@
-<div class="flex flex__align-items-center flex__justify-between margin-bottom__24px">
+<h1 class="font__12px font__weight-500 flex flex__align-items-center text__uppercase">
+	<span routerLink="/projects" class="font__weight-500 pointer-cursor">Projects</span>
+	<svg width="16" height="16" class="margin-x__4px">
+		<use xlink:href="#arrow-right-icon"></use>
+	</svg>
+	PROJECT NAME
+	<svg width="16" height="16" class="margin-x__4px">
+		<use xlink:href="#arrow-right-icon"></use>
+	</svg>
+	<span class="color__primary font__weight-500">Apps</span>
+</h1>
```

Comment:
Since this element is used everywhere, why not make it a component.

Reply:
Also for the span here, although `routerLink` does the logic, it still pays to use an accessible element.